src/app/workflow/workflow.component.html

```diff
@@ -265,3 +265,4 @@ <h3>DOI</h3>
       </div>
     </div> -->
   </div>
+</div>
```

Comment:
Was this missing a closing `</div>` all along?

Reply:
looks like it